language/diem-vm/src/diem_transaction_executor.rs

```diff
@@ -527,9 +533,9 @@ impl DiemVM {
         )
     }
 
-    pub fn execute_writeset_transaction(
+    pub fn execute_writeset_transaction<R: RemoteCache + StateView>(
```

Comment:
I intended to use this function as entrypoint for various transaction execution scheduler implementation